examples/info.rs

```diff
@@ -0,0 +1,33 @@
+#[macro_use]
+extern crate redis_module;
+
+use redis_module::{Context, NextArg, RedisError, RedisResult, RedisString, RedisValue};
+use std::borrow::Borrow;
+
+fn info_cmd(ctx: &Context, args: Vec<RedisString>) -> RedisResult {
+    if args.len() < 3 {
+        return Err(RedisError::WrongArity);
+    }
+
+    let mut args = args.into_iter().skip(1);
+
+    let section: RedisString = args.next_arg()?;
+    let field: RedisString = args.next_arg()?;
+
+    let server_info = ctx.server_info(section.borrow());
+    match server_info.field(field.borrow()) {
+        None => Ok(RedisValue::Null),
+        Some(v) => Ok(RedisValue::BulkRedisString(v)),
+    }
```

Comment:
1. There's no need for the `borrow`, just use `next_str` instead of `next_arg`.
2. When matching on builtin types such as `Option`, it's almost always more idiomatic to use combinators rather than a raw `match`.

```suggestion
    let section = args.next_str()?;
    let field = args.next_str()?;

    let server_info = ctx.server_info(section);

    Ok(server_info
        .field(field)
        .map_or(RedisValue::Null, RedisValue::BulkRedisString))
```